angular/projects/admin-nrpti/src/app/mines/mines-routing.module.ts

```diff
@@ -65,6 +66,35 @@ const routes: Routes = [
             resolve: {
               mine: MinesResolver
             }
+          },
+          {
```

Comment:
Good catch, I hadn't thought about that.  I am not 100% sure on the best approach for this, off the top of my head.  Possibly this might relate to the ticket about saving state?

Reply:
Yeah... the "lazy" approach is probably just creating a second route that links to the same component modules, but then we'll need a check when a user hits "update" to route back to the right place. I guess it depends on the workflow that's expected here... you're right though, we may need to save some state

I'm thinking the easiest approach is something kinda ugly like: Create a second route for records, then on edit we pass in a param on the URL or something for "where did I come from", so we can navigate back after an update click or on errors. Might want to have a discussion for it if not put the handling into a separate ticket; this will likely come up again with this stuff!

Reply:
Right, yeah I see what you're saying.  I think that might actually be the only possible option currently.  Because the breadcrumbs are based on the values in the routers, so if we ever wanted to the breadcrumb to say "mine > detail > edit record" it would need to be in that same chain of routes.  There might also be some way to re-configure the routing or the breadcrumbs component to handle this case better, but that sounds like a lot of work.

I do think I'd rather make a ticket for this though, so the other devs can add their 2 cents. Especially as this would require updates to every add-edit/details page on the admin site.

Reply:
Added a spike ticket: https://bcmines.atlassian.net/browse/NRPT-245